integration-tests/src/lib.rs

```diff
@@ -458,15 +459,15 @@ fn place_coretime_spot_order_from_para_to_relay() {
 	let beneficiary: sp_runtime::AccountId32 = [1u8; 32].into();
 
 	// Setup: reserve transfer from relay to Pop, so that sovereign account accurate for return transfer
-	let amount_to_send: Balance = pop_runtime::UNIT * 1000;
+	let amount_to_send: Balance = pop_runtime_devnet::UNIT * 1000;
```

Comment:
As mentioned elsewhere, maybe an alias of `pop_runtime_devnet` to `pop_runtime` to simplify?

Reply:
Addressed in a PR to this one.